### ISSUE TITLE
[Accessibility]: Increase height of bottom label to avoid truncation

### ISSUE THREAD

**Issue by darius-prince** [carbon-design-system/carbon-charts#1563]
### Name

Darius Wilson

### Are you an IBM employee?

- [X] Yes

### Environment

Browser

### What happened? What did you expect to happen instead?

My bottom labels are long and they are truncated by default.
I find this as an accessibility issue as the user will have trouble reading the labels

I didn't find a way to prevent the truncation or increase label height

Maybe a `truncate:false` or `width:70` option in `bottom` option

### What WCAG 2.1 checkpoint does the issue violate?

Readability

### Version

"@carbon/charts": "^1.6.1",
    "@carbon/charts-react": "^1.6.1",

### Data & options used

```js
axes: {
                    left: {
                        ticks: {
                            formatter: (value) => {
                                if (value >= 1000) {
                                    return (value / 1000) + "k";
                                }
                                return value;
                            }
                        },
                        title: "Vulnerable Item Count",
                        mapsTo: "value",
                        stacked: true,
                    },
                    bottom: {
                        mapsTo: "key",
                        scaleType: "labels",                     
                    }
                },
```

### What priority level would this issue be in your opinion?

P0

**Comment by darius-prince** [comment]
![image](https://github.com/carbon-design-system/carbon-charts/assets/133155663/4117b6a6-88fa-43e3-9372-3a4e1907cb05)


This is what I mean.
I can find a work around
